From d3c91e7271a9d64d23ef5b00e868cb68bc99644b Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 3 Oct 2016 17:04:11 -0700
Subject: [PATCH] TOOLS-1558 TOOLS-1555 broke sdcnode builds: 'npm install'
 isn't run, buildEnv changes break on spaces (fix Makefile syntax)

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2c6b520..39a1b64 100644
--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,7 @@ all: build/src nodes bits
 
 .PHONY: npmdeps
 npmdeps:
-    npm install
+	npm install
 
 build/src:
 	git clone https://github.com/nodejs/node.git build/src
-- 
2.21.0

